assets/css/style.css

```diff
@@ -456,3 +453,596 @@ footer a {
+    animation-name: slidein;
+    animation-duration: 3s;
+  }
+
+  @keyframes slidein {
+    from {
+      margin-left: 100%;
+      width: 300%;
+    }
+
+    to {
+      margin-left: 0%;
+      width: 100%;
+    }
+  }
+
+  .worktitle h1 br {
+    display: none;
+  }
+
+  .work .divider {
+    border: 1px solid #646369;
+  }
+
+  .work .cards {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+    width: 1156px;
+    margin: 88px 0;
+  }
+
+  .work .cards .card {
+    width: 1156px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto;
+    margin: 88px 0;
+    align-items: center;
+  }
+
+  .card .profile {
+    width: 684px;
+    height: 448px;
+    background: linear-gradient(0deg, #f9f9f9, #f9f9f9), url(image_processing20200716-21735-f07s4k.png);
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 0 24px;
+  }
+
+  .work .cards .card:nth-child(even) .profile {
+    order: 1;
+  }
+
+  .card .card-items {
+    display: flex;
+    flex-direction: column;
+    margin: 88px 0;
+    width: 448px;
+    height: 448px;
+    padding: 0;
+  }
+
+  .card .card-items .card-text {
+    width: 448px;
+    margin: 0 24px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .card .card-items .card-text .workText {
+    display: flex;
+    width: 448px;
+    flex-wrap: wrap;
+  }
+
+  .card .workText h3 {
+    width: 448px;
+    height: 40px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 44px;
+    color: #fff;
+    flex: none;
+    flex-grow: 0;
+    margin: 24px 0;
+  }
+
+  .card .workText p {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #dfe1e6;
+    margin: 24px 0;
+  }
+
+  .card .card-items .categories {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+    gap: 2px;
+    list-style: none;
+    padding: 0;
+    justify-content: space-between;
+    width: 307px;
+    margin: 24px 0;
+  }
+
+  .card .card-items .categories li {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    flex: none;
+    flex-grow: 0;
+    margin: 0 0;
+    background: #191765;
+  }
+
+  .card .categories li:not(:nth-child(1)) {
+    border-left: 1px solid #555;
+    height: 25px;
+    padding-left: 15px;
+  }
+
+  .btn {
+    align-self: flex-start;
+  }
+
+  .btn button {
+    position: static;
+    outline-style: none;
+    border-style: none;
+    width: 89px;
+    height: 48px;
+    left: 0;
+    top: 50px;
+    background: #7d73fc;
+    color: #fff;
+    flex: none;
+    flex-grow: 0;
+    margin: 24px 0;
+  }
+
+  .btn button:hover {
+    background: #dcdce8;
+    color: #555;
+  }
+
+  /* contact section */
+  .contact-sec {
+    display: flex;
+    flex-direction: column;
```

Comment:
- [ ] Kindly decrease the space between work section and contact section following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) . Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/160214768-1511d775-f00e-4f1e-913c-4ae1f70006e3.png)


Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/160214773-a80f2cfd-95a1-41b5-8043-a70e85399e9a.png)

Reply:
Applied suggested changes